Snippets/Sqs/Sqs_2/Usage.cs

```diff
@@ -8,6 +8,7 @@ class Usage
 
         var transport = endpointConfiguration.UseTransport<SqsTransport>();
         transport.Region("ap-southeast-2");
+        // bucket only required for messages larger than 256KB
```

Comment:
should the s3 bucket configuration be a dedicated section instead?

Reply:
Not sure if really required for a getting started

Reply:
> Not sure if really required for a getting started

that's what I meant. I'm not sure this needs to be in the getting started section at all?

Reply:
Left it as is. We can always remove it at a later stage if we think it is too complex for a getting started section.